src/sentry/static/sentry/app/views/eventsV2/index.tsx

```diff
@@ -32,10 +30,9 @@ import SavedQueryButtonGroup from './savedQuery';
 import EventView from './eventView';
 import EventInputName from './eventInputName';
 import {getFirstQueryString} from './utils';
-import {SAMPLE_VIEWS} from './data';
 import QueryList from './queryList';
 
-const DISPLAY_SEARCH_BAR_FLAG = false;
+const DISPLAY_SEARCH_BAR_FLAG = true;
```

Comment:
@doralchan search bar looks good to me. Can you please revert this back to `false` prior to merging?

Reply:
👍